src/elements/logo/logo.scss

```diff
@@ -24,6 +24,10 @@
   }
 }
 
+:host([hide-text]) {
```

Comment:
Let's discuss with Manuel as said, if we can accept a property on the sbb-logo for that case.